<launch>
    <param name="config_file" value="$(find o2ac_pose_distribution_updater)/launch/estimator_config.yaml"/>
    <rosparam>
    marker_array_topic_name: o2ac_pose_belief_visualization_marker_array
    visualization_scale: [1.0, 1.0, 1.0]
    mean_color: [1.0, 0.0, 0.0, 1.0]
    variance_color: [0.0, 0.0, 1.0, 0.04]
    number_of_particles_to_visualize: 50

    use_planning_scene_monitor: false
    visualize_each_step: true
    visualization_life_time: 1.2
    </rosparam>
    
    <node name="pose_distribution_updater" pkg="o2ac_pose_distribution_updater" type="update_distribution_action_server" output="screen">
    </node>
</launch>